packages/genesys-spark-components/src/components/beta/gux-cta-group/gux-cta-group.tsx

```diff
@@ -0,0 +1,126 @@
+import { Component, h, JSX, Prop, Element } from '@stencil/core';
+
+import { GuxCTAGroupAlignment } from './gux-cta-group.types';
+import { logWarn } from '@utils/error/log-error';
+
+@Component({
+  styleUrl: 'gux-cta-group.scss',
+  tag: 'gux-cta-group',
+  shadow: true
+})
+export class GuxCTAGroup {
+  /*
+   * Reference to the host element.
+   */
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * Sets the buttons alignment
+   */
+  @Prop()
+  align: GuxCTAGroupAlignment = 'start';
+  /**
+   * Defines if the primary button should have a danger accent
+   */
+  @Prop()
+  dangerous: boolean = false;
+
+  private validatePrimarySlot(
+    slottedElement:
+      | HTMLGuxButtonElement
+      | HTMLGuxActionButtonElement
+      | HTMLGuxButtonMultiElement
+      | HTMLGuxButtonSlotElement,
+    dangerous: boolean = false
+  ): void {
+    if (slottedElement) {
+      const slottedTagName = slottedElement.tagName;
+
+      if (
+        [
+          'GUX-BUTTON',
+          'GUX-ACTION-BUTTON',
+          'GUX-BUTTON-MULTI',
+          'GUX-BUTTON-SLOT'
+        ].includes(slottedTagName)
+      ) {
+        if (dangerous) {
+          slottedElement.accent = 'danger';
+        } else {
+          slottedElement.accent = 'primary';
+        }
+
+        return;
+      }
+
+      logWarn(this.root, `You must slot a button element in the primary slot.`);
+    }
+  }
+
+  private validateSecondarySlot(
+    slottedElement:
+      | HTMLGuxButtonElement
+      | HTMLGuxActionButtonElement
+      | HTMLGuxButtonMultiElement
+      | HTMLGuxButtonSlotElement
+  ): void {
+    if (slottedElement) {
+      const slottedTagName = slottedElement.tagName;
+
+      if (
+        ![
+          'GUX-BUTTON',
+          'GUX-ACTION-BUTTON',
+          'GUX-BUTTON-MULTI',
+          'GUX-BUTTON-SLOT'
+        ].includes(slottedTagName)
+      ) {
+        logWarn(
+          this.root,
+          `You must slot a button element in the secondary slot.`
+        );
+      }
```

Comment:
We should set the `accent` to secondary here. I know it is the default but incase the dev is setting it to primary to have two primary buttons or something.